### DISCUSSION TITLE
No way to unmark as Done

### DISCUSSION THREAD

**Discussion by jidanni** [community/community#78238] [Mobile]
If the user accidentally marks something as done, there is no way to unmark it as done. I'm talking about the mobile app.

**Comment by iskae** [comment]
Hi @jidanni 👋 

Thanks for reaching out ❤️ 

We are offering this capability through the first filter in the list 🎉 You can filter for the notifications that you have marked as done and mark the ones that you would like as undone through swipe actions or through long tapping on the notification and using multi-selection actions 😊 

Does that help you?

![Screenshot 2023-12-08 at 3 14 35 PM](https://github.com/community/community/assets/26570736/f7cd2cee-e647-443d-89ad-c4914f8e0f8d)

**Reply by jidanni** [reply]
Okay, thanks. And for anybody reading this, after a long press, in the upper right you will see a picture of a tray. Tap that to put the item back into the Inbox. In other words, unmark it as done.

As far as swipe actions, one should disable them as much as they can, because they're the ones that usually get one in trouble in the first place. (Usually a finger accidentally moved, causes a message to disappear.)
